packages/date/src/index.js

```diff
@@ -395,7 +395,7 @@ export function getAllowedIntervalsForQuery( query ) {
 				allowed = [ 'day', 'week', 'month', 'quarter' ];
 				break;
 			default:
-				allowed = [ 'hour', 'day' ];
+				allowed = [ 'day' ];
```

Comment:
This is working well.  Just thinking out loud: I'm wondering what periods hit here (just the "XX to date" presets?) and if we'd ever want to offer the hour as an option for the default case.

`allowed = [ 'day', 'hour' ];`

This would still default to `day` and allow hours.  Although my gut is telling me we might not want hours for those cases anyway.

Reply:
I think the only time this default clause should be hit is when the `query` object passed in as an argument has a `period` attribute that is either empty ( which is the default behavior when hitting any report or the dashboard it seems), or for some reason a period is passed in that is not in [this list](https://github.com/woocommerce/wc-admin/blob/master/packages/date/src/index.js#L37-L49). So I think the logic is pretty safe, ultimately as @LevinMedia stated in the issue, `hour` should only ever be an option when looking at `today` or `yesterday` in the predefined periods, or if looking at a day or less, which is handled in the `custom` period logic higher up.

I do think it would be worth adding in some more tests around the `custom` period handling above to make sure all is well there too.

Reply:
Okay that sounds similar to what I was thinking and seems logical.  LGTM! 🚀